add-interactive.c

```diff
@@ -7,6 +7,12 @@
 #include "revision.h"
 
 #define HEADER_INDENT "      "
+#define HELP_INFO "status        - show paths with changes\n"\
+		"update        - add working tree state to the staged set of changes\n"\
+		"revert        - revert staged set of changes back to the HEAD version\n"\
+		"patch         - pick hunks and update selectively\n"\
+		"diff          - view diff between HEAD and index\n"\
+		"add untracked - add contents of untracked files to the staged set of changes"

```

Comment:
This should probably be a `static const char *`. That would also avoid the backslashes at the end, and make compiler warnings/errors less opaque (in case anything breaks in the future).

Was this marked translatable in the Perl script? If so, we need to enclose it in `N_(...)` here, and use the constant enclosed in `_(...)` later on.